operator/roles/forkliftcontroller/templates/security/networkpolicy-migration-workloads.yml.j2

```diff
@@ -0,0 +1,280 @@
+# Network policies for migration workloads
+# Includes virt-v2v, consumer, and populator pods
+#
+# NFS Configuration:
+# - Default: NFSv4 (port 2049 TCP/UDP only)
+# - NFSv3: Set nfs_v3_support=true to enable ports 111 (rpcbind) and 20048 (mountd)
+# - Note: NFSv3 mountd port may vary depending on server configuration
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: forklift-virt-v2v-policy
+  namespace: {{ app_namespace }}
+  labels:
+    app: {{ app_name }}
+spec:
+  podSelector:
+    matchLabels:
+      forklift.app: virt-v2v
+  policyTypes:
+  - Ingress
+  - Egress
+  ingress:
+  # Allow controller communication
+  - from:
+    - podSelector:
+        matchLabels:
+          control-plane: controller-manager
+    # all ports allowed (ports omitted)
+  egress:
+  # Allow communication with controller
+  - to:
+    - podSelector:
+        matchLabels:
+          control-plane: controller-manager
+    # All ports allowed (ports omitted)
+  # Allow external source connections (vCenter, oVirt, etc.)
+  - ports:
+    - protocol: TCP
+      port: 443   # HTTPS
+    - protocol: TCP
+      port: 8443  # HTTPS alternate / oVirt Engine HTTPS
```

Comment:
8443 Dev not alternative

Reply:
Nope, that's not my point.
We should not be using development ports of other products.